Shared/Services/BackupsService.swift

```diff
@@ -97,6 +99,24 @@ class BackupsService: ObservableObject {
         }
         self.foldernames = folders
     }
+
+    func loadAllDevices() async {
+        do {
+            self.devices = try await DeviceService.shared.getAllDevices(deviceName: ConfigLoader().getDeviceName())
+        } catch {
```

Comment:
propagate errors, and catch them in the consumer, in this case, if you are going to use this from the `AppDelegate`, do the catch there, be careful with execution order, cause if one fails, the next one is not going to be executed